.travis.yml

```diff
@@ -31,30 +31,11 @@ env:
     - CI="true"
 python: "3.6"
 stages:
-  - pre-test
   - test
 services:
   - docker
 jobs:
   include:
```

Comment:
I am afraid we do not gain much by this. We have quite a lot of tests NOT running because static checks fail - that's probably >50% cases. If we just remove pre-test steps, then we will have a lot more "test" jobs  blocking the Travis Queue because they will run regardless if static checks fail.